### PR TITLE
[SYSTEMDS-3190] Do not run workflow for empty checks

### PR DESCRIPTION
**Pull request by j143**
```yml
on:
  push:
    paths-ignore:
      - 'docs/**'
      - '*.md'
      - '*.html'
      - 'src/main/python/docs/**'
      - 'dev/**'
```

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-ignoring-paths

### CODE REVIEW

File: .github/workflows/documentation.yml [hunk 1]
```diff
@@ -23,12 +23,21 @@ name: Documentation
...
+      - 'dev/**'
     branches:
       - master
   pull_request:
     paths-ignore:
       - 'docs/**'
       - '*.md'
+      - '*.html'
+      - 'src/main/python/docs/**'
+      - 'dev/**'
```
Comment (Baunsgaard) [inline .github/workflows/documentation.yml:40]: This is the one tests i would run on changes in docs, since it is testing if the docs build.

**Review by Baunsgaard — COMMENTED** [review]
LGTM except the documentation test.

File: .github/workflows/documentation.yml (hunk 1 above)
Comment (j143) [reply .github/workflows/documentation.yml:40]: Is this the change?

```diff
  pull_request:
    paths-ignore:
      - 'docs/**'
      - '*.md'
      - '*.html'
-      - 'src/main/python/docs/**'
      - 'dev/**'
```

or should not ignore any paths here?


Thanks

Comment (Baunsgaard) [reply .github/workflows/documentation.yml:40]: 'docs/**'
 'src/main/python/docs/**'
 and
 '*.md'
should not be ignored when we test building the docs.

**Pull request merged**